felix/bpf/ut/pol_prog_test.go

```diff
@@ -613,9 +952,166 @@ var polProgramTests = []polProgramTest{
 			tcpPkt("10.0.0.2:80", "10.0.0.1:31245"),
 			udpPkt("10.0.0.2:80", "10.0.0.1:31245")},
 	},
+	{
```

Comment:
The tests below are v6 specific tests or is it for mized 4 and 6?

Reply:
mixed IPv4 and 6. I noticed there is no test about /0 in IPv4 so I added it.